frontend/src/lib/candidate/components/input/InputField.type.ts

```diff
@@ -2,7 +2,11 @@ export type InputFieldProps<TValue extends AnswerProps['value']> = {
   /**
    * The question that is to be answered.
    */
-  question: QuestionProps;
+  questionId: string;
```

Comment:
Maybe we could rename this to just `id` in all of the inputs now that it's no longer tied to question? The comment could be changed to: The id of the input.

Reply:
I would leave it as questionId. Even though it does not have to be the question's id for the components, we still need the question's id to update the answers to the stores containing them. If this was named id, it would not be apparent that this still needs to be the question's id for the inputs to work correctly. Of course we could still do this, but then the logic of saving the answers would have to be refactored or we would need another attribute for the questionId.

Reply:
Okay, let's leave it as is and make any future changes in #580